src/rainfly/src/lib/actions/portal.js

```diff
@@ -0,0 +1,46 @@
+// REF: https://github.com/romkor/svelte-portal/blob/a650e7b762344a1bb0ad9e218660ed1ee66e3f90/src/Portal.svelte
```

Comment:
Do we need to include the license for this?

https://github.com/romkor/svelte-portal/blob/master/LICENSE